tardis/plasma/properties/atomic.py

```diff
@@ -94,11 +96,12 @@ class PhotoIonizationData(ProcessingPlasmaProperty):
     """
     Attributes
     ----------
-    photo_ion_cross_sections : Pandas DataFrame (nu, x_sect,
-                                                index=['atomic_number',
-                                                       'ion_number',
-                                                       'level_number']),
-                                                dtype float)
+    photo_ion_cross_sections : Pandas DataFrame
+        Table of
+        (nu, 
+        x_sect,
+        index=['atomic_number','ion_number','level_number']),
+        dtype float
```

Comment:
Same goes here - you can phrase this description in above format but do place it after description `... of frequency` in line 106.

Reply:
Same here?

Reply:
So just remove all the descriptors for what's inside the DataFrames?